### PR TITLE
[13.0][MIG] connector jira servicedesk

### PR DESCRIPTION
**Pull request by leemannd**

### CODE REVIEW

**Comment by grindtildeath** [comment]
Looks like we should still split the readme here

**Comment by guewen** [comment]
> Looks like we should still split the readme here

this + check runbot (maybe a warning?)

**Comment by leemannd** [comment]
@guewen @simahawk The build is fixed. Could you review it?
I will squash it once I have your approvals

**Review by simahawk — APPROVED** [review]
LG. Squash pls

File: connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot [hunk 1]
```diff
@@ -0,0 +1,161 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* connector_jira_servicedesk
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```
Comment (StephaneMangin) [inline connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot]: Maybe
```suggestion
"Project-Id-Version: Odoo Server 13.0\n"
```

Comment (StephaneMangin) [reply connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot]: Regenerate the whole translation just to be sure ^^

File: connector_jira_servicedesk/models/account_analytic_line/importer.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Camptocamp SA
```
Comment (StephaneMangin) [inline connector_jira_servicedesk/models/account_analytic_line/importer.py]:
```suggestion
# Copyright 2019-2021 Camptocamp SA
```

File: connector_jira_servicedesk/models/account_analytic_line/common.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Camptocamp SA
```
Comment (StephaneMangin) [inline connector_jira_servicedesk/models/account_analytic_line/common.py]:
```suggestion
# Copyright 2020-2021 Camptocamp SA
```

File: connector_jira_servicedesk/tests/test_import_organization.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Camptocamp SA
```
Comment (StephaneMangin) [inline connector_jira_servicedesk/tests/test_import_organization.py]:
```suggestion
# Copyright 2019-2021 Camptocamp SA
```

File: connector_jira_servicedesk/tests/common.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Copyright 2019 Camptocamp SA
```
Comment (StephaneMangin) [inline connector_jira_servicedesk/tests/common.py]:
```suggestion
# Copyright 2019-2021 Camptocamp SA
```

**Review by StephaneMangin — APPROVED** [review]
Pre-approval due to minor comments (maybe good to check for the test coverage btw)

File: connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot (hunk 1 above)
Comment (simahawk) [reply connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot]: not needed. POT will be re-generated

File: connector_jira_servicedesk/models/account_analytic_line/importer.py (hunk 2 above)
Comment (simahawk) [reply connector_jira_servicedesk/models/account_analytic_line/importer.py]: not needed, 2019 is the start

**Comment by leemannd** [comment]
@simahawk squashed. Ready to be merged

**Comment by leemannd** [comment]
Hello @simahawk , this is ready for your merge :)

**Comment by simahawk** [comment]
@leemannd likely you squashed too much on top of `Add servicedesk_issue_url` which contains many formatting changes...
Not a blocker. Just take into account next time :wink:

**Pull request merged**